cypress/support/pages/home.pageObject.js

```diff
@@ -4,12 +4,42 @@ class HomePageObject extends PageObject {
   url = '/#/';
 
   get usernameLink() {
-    return cy.getByDataCy('username-link');
+    return cy.get(':nth-child(4) > .nav-link');
```

Comment:
The CSS selector for `usernameLink` is quite specific and relies on the exact structure of the HTML. If the HTML structure changes, this selector might break. Consider using a more robust selector, such as a data attribute, if available.